src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntry.cmp

```diff
@@ -0,0 +1,165 @@
+
+                        <div class="slds-grid slds-grid_align-spread slds-p-bottom_x-small">
+                            <span>{!$Label.c.bgeGridGiftCountHeader}</span>
+                            <span>
+                                <strong>
+                                    <aura:if isTrue="{! !v.isNamespaced}">
+                                        {!format($Label.c.bgeGridTotalCount, v.totals.countGifts, v.record.Expected_Count_of_Gifts__c)}
+                                    </aura:if>
+                                    <aura:if isTrue="{!v.isNamespaced}">
+                                        {!format($Label.c.bgeGridTotalCount, v.totals.countGifts, v.record.npsp__Expected_Count_of_Gifts__c)}
+                                    </aura:if>
+                                </strong>
+                            </span>
+                        </div>
+                        <!--Progress bar is on a 100 point scale, so just using division doesn't work here-->
+                        <aura:if isTrue="{! !v.isNamespaced}">
+                            <lightning:progressBar value="{!mult(100,div(v.totals.countGifts, v.record.Expected_Count_of_Gifts__c))}"
+                                                   size="large"/>
+                        </aura:if>
+                        <aura:if isTrue="{!v.isNamespaced}">
+                            <lightning:progressBar value="{!mult(100,div(v.totals.countGifts, v.record.npsp__Expected_Count_of_Gifts__c))}"
+                                                   size="large"/>
+                        </aura:if>
+                    </lightning:layoutItem>
+
+                    <lightning:layoutItem size="6" padding="around-small">
+                        <label for="expectedTotalAmount" class="slds-p-right_xx-small">{!$Label.c.bgeGridTotalAmount}</label>
+                        <lightning:formattedNumber value="{!v.totals.totalGiftAmount}" style="currency" />
+                        /
+                        <aura:if isTrue="{! !v.isNamespaced}">
+                            <lightning:formattedNumber value="{!v.record.Expected_Total_Batch_Amount__c}" style="currency"/>
+                        </aura:if>
+                        <aura:if isTrue="{!v.isNamespaced}">
+                            <lightning:formattedNumber value="{!v.record.npsp__Expected_Total_Batch_Amount__c}" style="currency"/>
+                        </aura:if>
+                    </lightning:layoutItem>
+
+                </lightning:layout>
+                <!--End expected totals-->
+
+                <aura:if isTrue="{!notequals(v.data.length, 0)}">
+                    <div class="{!if(lessthan(v.data.length, 4), 'datatableContainer', '')+' slds-p-top_small'}">
+                        <lightning:datatable
+                                aura:id="dataImportRowsDataTable"
+                                columns="{! v.columns }"
```

Comment:
Is there intent behind the extra spaces around attributes? (`{! v.columns }` vs. `{!v.columns}`)

Reply:
Nope, just inconsistency. I like the space-less version better. We do add a space for the ! !v.isNamespaced just to make that easier to read, but the not() is better there anyways (if only a bit longer to type).